### ISSUE TITLE
Uninitialized constant Solargraph::Library::Pathname

### ISSUE THREAD

**Issue by ianmurrays** [castwide/solargraph#448]
Hi!

I'm getting an error (using vscode-solargraph and solargraph 0.42.2, on ruby 2.5), where the extension stalls while "Mapping Workspace" and with this error:

```
solargraph-0.42.2/lib/solargraph/library.rb:473:in `block (2 levels) in find_external_requires': uninitialized constant Solargraph::Library::Pathname
```

This line right here, it seems https://github.com/castwide/solargraph/blob/v0.42.2/lib/solargraph/library.rb#L473

Any idea if I'm doing something wrong myself?

Thanks!

**Comment by ianmurrays** [comment]
It would seem this is an issue with ruby 2.5, where one needs to `require 'pathname'`, whereas ruby 2.7 and above include it by default.

**Comment by castwide** [comment]
The `require` is safe in Ruby 2.7+, so I patched it in gem 0.42.3.

**Comment by ianmurrays** [comment]
Works now, thanks for the quick fix!
